nextpack/crates/sync-workspace/src/main.rs

```diff
@@ -0,0 +1,311 @@
+        .as_table()
+        .unwrap()
+        .clone();
+    remap_path_dependencies(&mut next_dependencies, TURBO_PATH);
+    remap_git_dependencies(
+        &mut next_dependencies,
+        "https://github.com/vercel/turbo.git",
+        &format!("{TURBO_PATH}crates/"),
+    );
+    let mut turbo_dependencies: Table = turbo_toml["workspace"]["dependencies"]
+        .as_table()
+        .unwrap()
+        .clone();
+    remap_path_dependencies(&mut turbo_dependencies, TURBO_PATH);
+
+    // Merge dependencies from Next.js and Turbo.
+    let mut conflicts_count = 0;
+    let mut dependencies = next_dependencies;
+    for (key, turbo_value) in turbo_dependencies {
+        match dependencies.entry(&key) {
+            Entry::Occupied(mut e) => {
+                let next_value = e.get_mut();
+                if *next_value != turbo_value {
+                    println!(
+                        "conflicting dependency:\nnext:  {} = {}\nturbo: {} = {}",
+                        key,
+                        get_diff_only(&next_value, &turbo_value),
+                        key,
+                        get_diff_only(&turbo_value, &next_value)
+                    );
+                    conflicts_count += 1;
+                }
+            }
+            Entry::Vacant(e) => {
+                e.insert(turbo_value);
+            }
+        }
+    }
+
+    // Ensure workspace path deps to members at least.
+    for member in members.iter() {
+        let dep = member.as_str().unwrap();
+        if let Some((path, name)) = dep.rsplit_once("/") {
+            if path == "crates" {
+                // ignore the nexpack helper crates.
+                continue;
+            }
+            if let Some(member_dep) = dependencies.get_mut(name).and_then(|d| d.as_table_mut()) {
+                member_dep.insert("path".to_string(), member.clone());
+            } else {
+                let mut member_dep = Table::new();
+                member_dep.insert("path".to_string(), member.clone());
+                dependencies.insert(name.to_string(), Value::Table(member_dep));
+            }
+        }
+    }
+
+    // Turbo's workspace dependencies must be duplicated for any of the crates to
+    // work, since they expect it to be in the workspace root. But we need to remap
+    // any path dependencies to be relative to Turbo's Cargo.toml.
+    cargo_toml["workspace"]
+        .as_table_mut()
+        .unwrap()
+        .insert("dependencies".to_string(), Value::Table(dependencies));
+
+    let mut toml_file = File::options()
+        .write(true)
+        .truncate(true)
+        .open("Cargo.toml")?;
+    writeln!(toml_file, "# THIS FILE IS AUTOGENERATED BY np-sync-crates")?;
+    writeln!(
+        toml_file,
+        "# Do NOT make changes to this file, instead change"
+    )?;
+    writeln!(toml_file, "# - {}/Cargo.toml, and", TURBO_PATH)?;
+    writeln!(toml_file, "# - {}/Cargo.toml", TURBO_PATH)?;
```

Comment:
This should be `NEXT_PATH`.

```suggestion
    writeln!(toml_file, "# - {}Cargo.toml", TURBO_PATH)?;
    writeln!(toml_file, "# - {}Cargo.toml", NEXT_PATH)?;
```

Also, need to remove the trailing /, otherwise we get the following:

```
# THIS FILE IS AUTOGENERATED BY np-sync-crates
# Do NOT make changes to this file, instead change
# - ./turbo-crates//Cargo.toml, and
# - ./turbo-crates//Cargo.toml
# and run `cargo run --bin sync-workspace`.
```